src/Header.js

```diff
@@ -93,16 +124,22 @@ class Header extends Component {
 			});
 			appBarStyle = css(AppStyles.headerBorderDetail, statusColor.color);
 		}
+		let notificationMessage = this.getNotificationMessage();
 
 		iconElementRight = <div>{this.getHelpButton()}{iconElementRight}</div>;
 
 		return (
-			<AppBar
-				title={<span>{this.props.title}</span>}
-				iconElementLeft={iconElementLeft}
-				iconElementRight={iconElementRight}
-				className={appBarStyle}
-			/>
+			<div>
+				<AppBar
+					title={<span>{this.props.title}</span>}
+					iconElementLeft={iconElementLeft}
+					iconElementRight={iconElementRight}
+					className={appBarStyle}/>
+				<Notification

```

Comment:
Please add an explaining comment or (better) put the inline js into own methods with meaningful names